pandas/tests/frame/conftest.py

```diff
@@ -127,6 +127,17 @@ def timezone_frame():
     return df
 
 
+@pytest.fixture
+def uint64_frame():
+    """
```

Comment:
instead of this can you just use the mixed_int_frame (maybe even add a column to it if needed)

Reply:
Nope, not changing the semantics of tests here. Just translating.